index.md

```diff
@@ -10,18 +10,18 @@ has_toc: false
 platform: false
```

Comment:
@DanHealey Please make sure to merge this with the latest changes that Manolo did recently (integrating YouTube video)